tests/test_pylint_runners.py

```diff
@@ -18,7 +19,7 @@
     "runner", [run_epylint, run_pylint, run_pyreverse, run_symilar]
 )
 def test_runner(runner: Callable, tmpdir: LocalPath) -> None:
-    filepath = os.path.abspath(__file__)
+    filepath = __file__
```

Comment:
```suggestion
    filepath = os.path.abspath(__file__)
```
Let's focus just on the typing change here.